src/lib.rs

```diff
@@ -38,6 +38,12 @@ impl Script {
         Self::inner_from_short_name(input)
     }
 
+    /// The 4-byte iso15924 tag as a `u32`
```

Comment:
nit: document as big-endian?

Reply:
Done!

Reply:
Hmm... yeah I think so. Perhaps this API ought return `[u8; 4]` to avoid ambiguity.